Formula/pyrsia.rb

```diff
@@ -12,7 +12,57 @@ def install
     bin.install "pyrsia_node"
   end
 
+  def post_install
+    (var/"pyrsia").mkpath
```

Comment:
Is it possible not to hardcode `var`?

Reply:
Here the `var` is defined by Homebrew. You may refer to [Variables for directory locations](https://docs.brew.sh/Formula-Cookbook#variables-for-directory-locations).